src/components/pages/Fuse/FusePoolEditPage.tsx

```diff
@@ -37,6 +36,7 @@ import BigNumber from "bignumber.js";
 import { useTokenData } from "../../../hooks/useTokenData";
 import LogRocket from "logrocket";
 import { handleGenericError } from "../../../utils/errorHandling";
+import useAuthedCallback from "../../../hooks/useAuthedCallback";
```

Comment:
Can we use nondefault imports if possible? Autocomplete works better with them and i'd like to follow that practice if possible 😅